src/ucs/sys/socket.c

```diff
@@ -198,25 +198,22 @@ const char* ucs_sockaddr_str(const struct sockaddr *sock_addr,
     size_t str_len;
 
     if ((sock_addr->sa_family != AF_INET) && (sock_addr->sa_family != AF_INET6)) {
-        ucs_error("unknown address family: %d", sock_addr->sa_family);
         ucs_strncpy_zero(str, "<invalid address family>", max_size);
         return str;
     }
 
     if (!inet_ntop(sock_addr->sa_family, ucs_sockaddr_get_inet_addr(sock_addr),
                    str, max_size)) {
-        ucs_error("inet_ntop failed: %m");
         ucs_strncpy_zero(str, "<failed to convert sockaddr to string>", max_size);
         return str;
     }
 
-    str_len = strlen(str);
-
     if (ucs_sockaddr_get_port(sock_addr, &port) != UCS_OK) {
-        ucs_strncpy_zero(str, "<unable to get port>", max_size);
```

Comment:
why removed? we still need to copy the error message to "str"

Reply:
yes, it was by mistake